src/services/programService/utils/authorizationHeader.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 The Ontario Institute for Cancer Research. All rights reserved
+ *
+ * This program and the accompanying materials are made available under the terms of
+ * the GNU Affero General Public License v3.0. You should have received a copy of the
+ * GNU Affero General Public License along with this program.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+ * EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES
+ * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT
+ * SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED
+ * TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+ * OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER
+ * IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ * ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/*
+ * This file dynamically generates a gRPC client from Ego.proto.
+ * The content of Ego.proto is copied directly from: https://github.com/icgc-argo/argo-proto/blob/4e2aeda59eb48b7af20b462aef2f04ef5d0d6e7c/ProgramService.proto
+ */
+
+// Create a function that return this repetitive string template in the fecth header
+const authorizationHeader = (jwt: string) => `Bearer ${jwt}`;
```

Comment:
You should write this comment as a JSDoc:

```ts
/**
 * Format JWT as Bearer Token for Authorization header.
 * @param {string} jwt 
 * @returns {string}
 * 
 * @example
 * ```
 * await fetch(url, {
 * 	method: 'get',
 * 	headers: {
 * 		Authorization: authorizationHeader(jwt),
 * 	},
 * })
 * ```
 */
const authorizationHeader = (jwt: string) => `Bearer ${jwt}`;

```

Reply:
JS Doc Reference: https://jsdoc.app/about-getting-started

Also, you don't need to specify the variable types in JSDoc when using TypeScript, since we trust the TS types not the types in the docs.